testkit/src/main/scala/org/bitcoins/testkit/node/NodeUnitTest.scala

```diff
@@ -58,19 +58,12 @@ trait NodeUnitTest extends BitcoinSFixture with EmbeddedPg {
   }
 
   override def afterAll(): Unit = {
-    Await.result(config.chainConf.stop(), 1.minute)
-    Await.result(config.nodeConf.stop(), 1.minute)
-    Await.result(config.walletConf.stop(), 1.minute)
```

Comment:
I think if we remove these the db threads will stay active and slow things down

Reply:
It would appear so, but actually this is is a method not a `val`. I am for renaming the `config` method to `getFreshConfig()` or something like that indicate that this is a new config, not an existing one  associated with the test suite.

https://github.com/christewart/bitcoin-s-core/blob/defdf6b0e0726dd58c0eea21a83241850ae67a78/testkit/src/main/scala/org/bitcoins/testkit/node/NodeUnitTest.scala#L65

Reply:
Good catch, ACK on that

Reply:
https://github.com/bitcoin-s/bitcoin-s/issues/2493 

I'm not going to address on this PR